src/Compilers/CSharp/Portable/Symbols/FieldSymbol.cs

```diff
@@ -420,6 +420,18 @@ public virtual FieldSymbol TupleUnderlyingField
         }
 
         /// <summary>
+        /// If this field represents a tuple element, returns a corresponding default element field.
+        /// Otherwise returns null
+        /// </summary>
+        public virtual FieldSymbol CorrespondingTupleField
```

Comment:
I used the style as in the rest of this file. I do not think it worth it changing everything and changing just this one would be inconsistent

---
In reply to: [86480154](https://github.com/dotnet/roslyn/pull/14958#discussion_r86480154) [](ancestors = 86480154)